app/templates/transformation_select.html

```diff
@@ -0,0 +1,32 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+    <h1>Select Image and Transformation</h1>
+    <form action="" method="post" novalidate>
+        {{ form.hidden_tag() }}
+        <h4>
+            Image:
+        </h4>
+        <p>
+            {{ form.image }}
+        </p>
+        <h4>
+            Parameters:
+        </h4>
+        <p>
+            Color : {{ form.color(min='0', max='10',step='0.1', value='1') }}
+        </p>
+        <p>
+            Brightness : {{ form.brightness(min='0', max='10',step='0.1', value='1') }}
+        </p>
+        <p>
+            Contrast : {{ form.contrast(min='0', max='10',step='0.1', value='1') }}
+        </p>
+        <p>
+            Sharpness : {{ form.sharpness(min='0', max='10',step='0.1', value='1') }}
```

Comment:
I think sharpness goes from 0 to 2 in the function you used. The others go from 0 to 1. Maybe you can still use the range that you prefer in the frontend, but you should rescale it before using it in the backend function. Or you can set the correct values here.